velox/common/memory/SharedArbitrator.cpp

```diff
@@ -516,8 +516,11 @@ uint64_t SharedArbitrator::reclaimUsedMemoryFromCandidatesByAbort(
     }
     try {
       VELOX_MEM_POOL_ABORTED(fmt::format(
-          "Memory pool aborted to reclaim used memory, current usage {}",
-          succinctBytes(candidate.currentBytes)));
+          "Memory pool aborted to reclaim used memory, current usage {}, "
+          "memory pool details:\n{}\n{}",
```

Comment:
Do wee need memory pool details in error log?

Reply:
There is no error log printed in this case. We just fail the query. I think it's enough to see the usage in failed query message. Adding error log here might pollute the log as the tree log is generally very long..